Documentation/cli/README.md

```diff
@@ -339,10 +348,11 @@ If regex is specified only the source files matching it will be returned.
 ## stack
 Print stack trace.
 
-	[goroutine <n>] [frame <m>] stack [<depth>] [-full] [-g] [-s] [-offsets]
+	[goroutine <n>] [frame <m>] stack [<depth>] [-full] [-offsets] [-defer]
 
 	-full		every stackframe is decorated with the value of its local variables and arguments.
-	-offsets	prints frame offset of each frame
+	-offsets	prints frame offset of each frame.
+	-defer	prints calls deferred by each frame.
```

Comment:
nit: let's reword this a bit, something like: `prints deferred function call stack for each frame`.

Reply:
Done.